holopy/inference/model.py

```diff
@@ -535,20 +510,6 @@ def _lnlike(self, pars, data):
             0.5 * (self._residuals(pars, data, noise_sd)**2).sum())
         return log_likelihood
 
-    def fit(self, data, strategy=None):
```

Comment:
I think you could have `model.fit()` and `model.sample()` methods `raise_fitting_api_error`. My rationale is that they don't really require any other adjustments and it's easy for someone to delete the few lines of code later so the cost of keeping them is minimal and the benefit is better clarity for anybody trying to use them but it's up to you.